src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -194,7 +194,7 @@ def __init__(self, es_search, manifest_entries, mapping, format_, object_key=Non
         The constructor takes the raw response from ElasticSearch and creates
         a csv file based on the columns from the manifest_entries
         :param es_search: The Elasticsearch DSL Search object
-        :param manifest_entries: The columns that will be present in the tsv
+        :param manifest_entries: The columns that will be present in the compact
```

Comment:
Search/replace error.
```suggestion
        :param manifest_entries: The columns that will be present in the manifest
```